src/Ushahidi/Modules/V5/Http/Resources/Permissions/PermissionsResource.php

```diff
@@ -11,6 +12,23 @@ class PermissionsResource extends Resource
 
     public static $wrap = 'data';
 
+
+    private function getResourcePrivileges()
+    {
+        $authorizer = service('authorizer.permission');
```

Comment:
Is there a way to call the Authorizer without using the service helper function, as this uses Aura DI, which we want to remove in the future?

Reply:
absolutely we will find another way, now I didn't find but I need to return this value, so the client side can build their SDK